src/dir_contents.ml

```diff
@@ -290,11 +297,12 @@ end = struct
       let lib_name = Lib_name.Local.to_string lib.name in
       if not (Library.Wrapped.to_bool lib.wrapped) ||
          (Module.Name.Map.cardinal modules = 1 &&
-          Module.Name.Map.mem modules main_module_name) then
+          Module.Name.Map.mem modules main_module_name) ||
+         Option.is_some lib.stdlib then
         None
       else if Module.Name.Map.mem modules main_module_name then
-        (* This module needs an implementation for non-jbuilder
-           users of the library:
+        (* This module needs an implementation for non-jbuilder users
```

Comment:
Maybe a typo, do you mean 'non-dune' ?

Reply:
Indeed, this is an old comment. I updated it